public/locales/en-US/common.json

```diff
@@ -17,22 +17,26 @@
 			"communityText": "Join our community"
 		}
 	},
-	"cookiesAlertModal": {
-		"alert": "This website uses cookies to ensure you get the best experience on our website.",
-		"confirmation": "Got It!"
-	},
 	"feedbackSection": {
+		"description": {
+			"title": "Don't take our word for it",
+			"subtitle": "See what our members have to say about us."
+		},
 		"feedbacks": {
 			"alessandroRamos": "The community is very welcoming, I was always afraid to ask questions in other communities for fear of being \"too stupid questions\", but at Techmmunity, people understand that nobody is born knowing, answer even the stupid questions without making fun of whoever is asking. It's cool to be able to ask questions without fear.",
 			"diegoPinheiro": "I have been to several technology communities to seek interaction to share experiences, but none of them met my expectations like the Techmmunity. Organized and with a visionary sense, Techmmunity seeks to integrate people from different areas to produce knowledge and add value in the form of projects, thus enabling better qualifications to obtain a solid position in the job market.",
 			"plactronic": "If you are looking for a community to learn, be it an exercise of a Bootcamp, college, work or want to understand why your code is returning a NaN or NullPointerException, it arrived on the right server. Always growing and with programmers working in the field for more than 5, 10 years and / or studying at an advanced level the languages they use, Techmmunity is your 127.0.0.1 to increase your networking and progress at the same time."
-		},
-		"description": {
-			"title": "Don't take our word for it",
-			"subtitle": "See what our members have to say about us."
 		}
 	},
 	"madeOfSection": {
+		"description": {
+			"title": "With space for everyone hungry for knowledge",
+			"subtitle": {
+				"subtitleBegin": "What is ",
+				"subtitleTechmmunity": "<COMPANY_NAME>",
+				"subtitleEnd": " made of?"
+			}
+		},
```

Comment:
remover os objetos de dentro da `subtitle` e atribuir o valor dela para `"What is <COMPANY_NAME> made of?"`